### PR TITLE
add S3 datasource doc

### PR DESCRIPTION
**Pull request by blotus**

### CODE REVIEW

File: crowdsec-docs/docs/data_sources/s3.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+
+### `polling_method`
+
+How to detect new files in a bucket. Must be one of `sqs` or `list`.
+`sqs` is the recommended mode.
+
+### `polling_interval`
+
+How often in seconds crowdsec will check for new objets in a bucket when using the `list` polling method.
```
Comment (he2ss) [inline crowdsec-docs/docs/data_sources/s3.md:48]: s/objets/objects

File: crowdsec-docs/docs/data_sources/s3.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+### `bucket_name`
+
+Name of the bucket to poll.
+Required when `polling_method` is `list`.
+
+
+### `prefix`
+
+Only read objets matching this prefix when `polling_method` is `list`.
```
Comment (he2ss) [inline crowdsec-docs/docs/data_sources/s3.md:64]: s/objets/objects/

**Review by buixor — APPROVED** [review]
lgtm

File: crowdsec-docs/docs/data_sources/s3.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+Optional, ignored when `polling_method` is `sqs`.
+
+### `aws_profile`
+
+The AWS profile to use, relies on your `~/.aws/config/`.
+
+Optional, the data source will automatically the standard AWS env vars if present.
```
Comment (he2ss) [inline crowdsec-docs/docs/data_sources/s3.md:71]: will automatically use

Comment (he2ss) [inline crowdsec-docs/docs/data_sources/s3.md:71]: will automatically use

File: crowdsec-docs/docs/data_sources/s3.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+### `aws_region`
+
+The AWS region.
+
+Optional, the data source will automatically the standard AWS env vars if present.
```
Comment (he2ss) [inline crowdsec-docs/docs/data_sources/s3.md:77]: automatically use

**Pull request merged**